tests/nodejs_test.go

```diff
@@ -1073,6 +1073,51 @@ func TestAccEksAutoModeUpgrade(t *testing.T) {
 	programTestWithExtraOptions(t, &test, nil)
 }
 
+func TestAccEfa(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+
+	// this is one of the cheapest instances that support EFA
+	instanceType := "g6.8xlarge"
```

Comment:
This test will cost roughly 1$ to run.
`g6.8xlarge`: $2.014 per hour, we run 2 of them for ~15 minutes